### ISSUE TITLE
[Beta1] Error - failed to create Default

### ISSUE THREAD

**Issue by Kestatwala** [TanninOne/modorganizer#694]
When I try to launch MO2 after installing it, I got the splash and an error messagebox saying:
"failed to create Default".

Clicking OK or closing the textbox stop MO2. So basically, I can't even launch it.

Tried with UAC disabled, and the ssleay.dll thing, but this didn't help.

Could it be that my user name contain a non-english character ("é") ? I regularly have problems with that.

Edit: 
Tried re-installing with various different set of plugins. Didn't help either :/
Testing on a other user account once I get back home.

**Comment by WhiteZeroX** [comment]
Try the solution here #620

**Comment by Kestatwala** [comment]
Damn, I was sure I checked all of the recent issues to avoid duplicate. Thanks.

Didn't work at first, but pretty much as the other guy explained, after randomly deleting things here and there, uninstalling, re-installing and all that a few times, it finally launched. I also experienced for the first time "Explorer as stopped working" in the process.

**Comment by kstarler** [comment]
As an aside for anyone else with this issue, I had this problem, and it turned out to be user error. I had installed MO to a folder named "ModOrganizer," but I had configured MO to use folders in "Mod Organizer" with a space.

I fixed it by going into the app data folder as indicated in [url=https://github.com/TanninOne/modorganizer/issues/620]#620[/url] and changing the MO .ini file to point to the correct paths.
